paseos/geometric_model/geometric_model.py

```
@@ -0,0 +1,102 @@
+        scale value. The default without scale values is a cube with 1m sides. This uses the python module Trimesh.
+
+        Returns:
+            mesh: Trimesh mesh of the satellite
+        """
+        if self.vertices is None:
+            # Defines the corners of the mesh, values are in meters, from the origin of the body frame.
+            self.vertices = [
+                [-0.5, -0.5, -0.5],
+                [-0.5, -0.5, 0.5],
+                [-0.5, 0.5, -0.5],
+                [-0.5, 0.5, 0.5],
+                [0.5, -0.5, -0.5],
+                [0.5, -0.5, 0.5],
+                [0.5, 0.5, -0.5],
+                [0.5, 0.5, 0.5],
+            ]
+            # List of three vertices to form a triangular face of the satellite.
+            # Two triangular faces are used per side of the cuboid.
+            self.faces = [
+                [0, 1, 3],
+                [0, 3, 2],
+                [0, 2, 6],
+                [0, 6, 4],
+                [1, 5, 3],
+                [3, 5, 7],
+                [2, 3, 7],
+                [2, 7, 6],
+                [4, 6, 7],
+                [4, 7, 5],
+                [0, 4, 1],
+                [1, 4, 5],
+            ]
+
+        mesh = trimesh.Trimesh(self.vertices, self.faces)
+        # Scales the mesh by the scale factor.
+        self._actor_mesh = mesh.apply_scale(self.scale)
+        return self._actor_mesh
+
+    @property
+    def find_moment_of_inertia(self):
+        """Gives the moment of inertia of the actor, assuming constant density.
+
+        Returns:
+            np.array: Mass moments of inertia for the actor
+
+        I is the moment of inertia, in the form of [[Ixx Ixy Ixz]
+                                                    [Iyx Iyy Iyx]
+                                                    [Izx Izy Izz]]
+        """
+        self._actor_moment_of_inertia = self._actor_mesh.moment_inertia
+        return self._actor_moment_of_inertia
```

this function doesn't do anything right now, so please raise a NotImplemented error so it's clear that nothing happens